src/gui/Src/Disassembler/ZydisTokenizer.h

```diff
@@ -172,6 +172,7 @@ class ZydisTokenizer
 
 
 private:
+    bool isSpaceTokenType(TokenType type);

```

Comment:
This should probably just be a `static tokenIsSpace(TokenType type)` in `ZydisTokenizer.cpp`, since it doesn't access the members at all

Reply:
removed `isSpaceTokenType` form `ZydisTokenizer.h` and left a `static` method `tokenIsSpace` in `ZydisTokenizer.cpp`